src/scripts/content/airtable.js

```diff
@@ -0,0 +1,22 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+
+'use strict';
+
+togglbutton.render('.detailViewWithActivityFeedBase .dialog > .header > .flex-auto:not(.toggl)', {observe: true}, function (elem) {
+
+  var link, descFunc,
+    container = $('.justify-center > .relative', elem),
+    description = $('.truncate.flex-auto.line-height-3', elem);
```

Comment:
Sadly those elements don't have more deterministic class names, so I'd go with perhaps `:nth-child` selectors. But probably not that much more reliable anyway.
These work for current interface layout, so this is good to go :)